### PR TITLE
Add `/_admin/metrics/v2` DocuBlock

### PR DESCRIPTION
**Pull request by Simran-B**
### Scope & Purpose

Document new Metrics endpoint, add deprecation note to old.

#### Backports:

- [x] Backports required for: 3.7, 3.9.devel

#### Related Information

- [x] Docs PR: https://github.com/arangodb/docs/pull/712

### Testing & Verification

*(Please pick either of the following options)*

- [x] This change is a trivial rework / code cleanup without any test coverage.
- [ ] The behavior in this PR was *manually tested*

### CODE REVIEW

**Comment by Simran-B** [comment]
Local preview:

image

File: Documentation/DocuBlocks/Rest/Administration/get_admin_metrics_v2.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+server will reply. This parameter is only meaningful on Coordinators.
+
+@RESTDESCRIPTION
+Returns the instance's current metrics in Prometheus format. The
+returned document collects all instance metrics, which are measured
+at any given time and exposes them for collection by Prometheus.
+
+The document contains different metrics and metrics groups dependent
+on the role of the queried instance. All exported metrics are
+published with the prefix `arangodb_` to distinguish them from other
```
Comment (neunhoef) [inline Documentation/DocuBlocks/Rest/Administration/get_admin_metrics_v2.md:20]: This is not 100% correct. We have quite a few which are prefixed with `rocksdb_` and do not have the `arangodb_` prefix. I suggest we add "or `rocksdb_`" to this sentence.

**Review by neunhoef — APPROVED** [review]
Tiny nitpicking comment and suggested addition, otherwise: LGTM.

File: Documentation/DocuBlocks/Rest/Administration/get_admin_metrics_v2.md (hunk 1 above)
Comment (KVS85) [reply Documentation/DocuBlocks/Rest/Administration/get_admin_metrics_v2.md]: Fixed.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-ANY-examples/838/

**Pull request merged**